app/components/title-input.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  classNames: ['input-edit'],
+  classNameBindings: ['active'],
+
+  //proxy clicks to input focus
+  click() {
+    this.$('input').focus();
+  },
+
+  actions: {
+    inputFocusIn() {
+      this.set('active', true);
+    },
+
+    inputFocusOut() {
+      this.set('active', false);
+    }
+  }
```

Comment:
Instead of setting your own class with these actions, you could have used the :focus pseudo-class in CSS.